supervision/draw/utils.py

```diff
@@ -111,6 +111,8 @@ def draw_text(
     text_padding: int = 10,
     text_font: int = cv2.FONT_HERSHEY_SIMPLEX,
     background_color: Optional[Color] = None,
+    text_width: Optional[int] = None,
```

Comment:
The input/output value to draw above and below the line zone, I had to update the y value of the text_anchor with text_offset and text_height. I made this change to avoid recalculation. I reverted my changes. Thanks for your advice